packages/persist/lib/server.integration.test.ts

```diff
@@ -101,23 +98,18 @@ describe('Persist API', () => {
             );
             expect(response.status).toEqual(400);
             const respBody = (await response.json()) as any[];
-            expect(respBody).toMatchObject([
-                {
-                    type: 'Body',
-                    errors: {
-                        issues: [
-                            {
-                                code: 'too_small',
-                                minimum: 1,
-                                type: 'array',
-                                message: 'Array must contain at least 1 element(s)',
-                                path: ['records']
-                            }
-                        ],
-                        name: 'ZodError'
-                    }
+            expect(respBody).toMatchObject({
+                error: {
+                    code: 'invalid_request',
+                    errors: [
+                        {
+                            code: 'too_small',
+                            message: 'Array must contain at least 1 element(s)',
+                            path: ['records']
+                        }
+                    ]
```

Comment:
the formatting of the errors changed but since the API is not exposed our customers it is not a breaking change